polkit.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: polkit
+  version: "126"
+  epoch: 0
+  description: "A toolkit for defining and handling authorizations."
+  copyright:
+    - license: LGPL-2.0-or-later
+  dependencies:
+    runtime:
+      - dbus
+      - linux-pam
+      - systemd
+      - wolfi-base
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - dbus-dev
+      - duktape
+      - expat-dev
+      - gettext-dev
+      - glib-dev
+      - glib-gir
+      - gobject-introspection-dev
+      - linux-pam-dev
+      - meson
+      - perl
+      - systemd-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: d627b0d1e1108563658dabe3fb8d2a065e64df10
+      repository: https://github.com/polkit-org/polkit.git
+      tag: ${{package.version}}
+
+  - uses: meson/configure
+    with:
+      opts: |
+        -Dman=true \
+        -Dpam_prefix=/etc/pam.d \
+        -Dpam_module_dir=/usr/lib/security
+
+  - uses: meson/compile
+
+  - uses: meson/install
+
+subpackages:
+  - name: polkit-doc
+    pipeline:
+      - uses: split/manpages
+    description: polkit manpages
```

Comment:
we need a few more subpackage splits here.

we need to do split-dev such that all the .pc files are split.
Also we need to split locales/translations too.
As otherwise the package is heavy.

To ensure polkit starts in priviledged containers with systemd, preset snippet is needed too please.

something to ponder if we want to split runtime libraries out as well, or not. Likely not worth it.

Reply:
Thanks! Done.